elfi/visualization/visualization.py

```diff
@@ -69,6 +70,114 @@ def nx_draw(G, internal=False, param_names=False, filename=None, format=None):
     return dot
 
 
+def plot_acq_points(points_acq, names_param, n_params, n_bins=20):
```

Comment:
I think this is essentially the same as the `plot_pairs` function below. Could you combine these so that we could avoid duplicate code? I think this is good then!